packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -365,6 +411,13 @@ export interface ClusterAttributes {
    * @default - No default namespace
    */
   readonly defaultCloudMapNamespace?: cloudmap.INamespace;
+
+  /**
+   * Getter for autoscaling group added to the cluster
```

Comment:
"Getter"?

Reply:
If the autoscaling group is created in the constructor, there needs to be a way to get it. (The AddCapacity method returns the ASG)

Reply:
Updated all comments to remove Getter terminology